### PR TITLE
Updated instructions in README to outline steps for using released VMC provider.

### PR DESCRIPTION
**Pull request by saloneerege**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,34 +9,118 @@ Requirements
...
+Since Terraform has no indication of version, it cannot upgrade in a native way, based on the “version” attribute in provider block.
+In addition, this may cause difficulties in housekeeping and issue reporting.
+
+
+## Cloning the Project
 
 The instructions outlined below to build the provider are specific to Mac OS or Linux OS only.
 
-Clone repository to: `$GOPATH/src/github.com/provider/`
+If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (please check the [requirements](https://github.com/terraform-providers/terraform-provider-aws#requirements) before proceeding).
```
Comment (shylasrinivas) [inline README.md]: The GitHub link is referencing AWS?

File: README.md [hunk 2]
```diff
@@ -9,34 +9,118 @@ Requirements
...
+**NOTE:** Before you start work on a feature, please make sure to check the
+[issue tracker][gh-issues] and existing [pull requests][gh-prs] to ensure that
+work is not being duplicated. For further clarification, you can also ask in a
+new issue.
+
+[gh-issues]: https://github.com/terraform-providers/terraform-provider-vmc/issues
+[gh-prs]: https://github.com/terraform-providers/terraform-provider-vmc/pulls
+
+If you wish to work on the provider, you'll first need [Go][go-website]
+installed on your machine (version 1.11+ is **required**). You'll also need to
```
Comment (shylasrinivas) [inline README.md]: Line #10 above mentions go version 1.13, and here it is 1.11+. Are we trying to say that any Go version above 1.11 is supported?

File: README.md [hunk 3]
```diff
@@ -9,34 +9,118 @@ Requirements
 - [Terraform](https://www.terraform.io/downloads.html) 0.12+
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
-## Build the Provider
+
+# Using the Provider
+
+To use a released provider in your Terraform environment, run [`terraform init`](https://www.terraform.io/docs/commands/init.html) and Terraform will automatically install the provider. To specify a particular provider version when installing released providers, see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#version-provider-versions).
+
+To instead use a custom-built provider in your Terraform environment (e.g. the provider binary from the build instructions above), follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-plugins) After placing the custom-built provider into your plugins directory,  run `terraform init` to initialize it.
```
Comment (rchadalawada) [inline README.md:42]: Where are the build instructions that you referred to in this line?

Comment (saloneerege) [reply README.md:42]: I need to edit that to point to the build instructions under building the provider.

Comment (saloneerege) [reply README.md:42]: Done

**Review by rchadalawada — COMMENTED** [review]
You check on the headings font or level. The building provider seems to be a sub-title

File: README.md [hunk 4]
```diff
@@ -9,34 +9,118 @@ Requirements
 - [Terraform](https://www.terraform.io/downloads.html) 0.12+
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
-## Build the Provider
 
-The instructions outlined below to build the provider are specific to Mac OS or Linux OS only.
+## Cloning the Project
```
Comment (rchadalawada) [inline README.md]: Make it a top level heading to be consistent

File: README.md [hunk 5]
```diff
@@ -9,34 +9,118 @@ Requirements
...
-mkdir -p $GOPATH/src/github.com/provider/
-cd $GOPATH/src/github.com/provider/
-git clone https://github.com/terraform-providers/terraform-provider-vmc.git
+mkdir -p $GOPATH/src/github.com/terraform-providers
+cd $GOPATH/src/github.com/terraform-providers
+git clone git@github.com:terraform-providers/terraform-provider-vmc.git
 ```
 
-Enter the provider directory and build the provider
+## Building the Provider
```
Comment (rchadalawada) [inline README.md]: Make it a top level heading to be consistent

File: README.md [hunk 6]
```diff
@@ -9,34 +9,118 @@ Requirements
...
+# Using the Provider
+
+To use a released provider in your Terraform environment, run [`terraform init`](https://www.terraform.io/docs/commands/init.html) and Terraform will automatically install the provider. To specify a particular provider version when installing released providers, see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#version-provider-versions).
+
+To instead use a custom-built provider in your Terraform environment (e.g. the provider binary from the build instructions above), follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-plugins) After placing the custom-built provider into your plugins directory,  run `terraform init` to initialize it.
+
+For either installation method, documentation about the provider specific configuration options can be found on the [provider's website](https://www.terraform.io/docs/providers/vmc/index.html).
+
+
+### Controlling the provider version
```
Comment (rchadalawada) [inline README.md]: this title is two level below the top level

File: README.md [hunk 7]
```diff
@@ -9,34 +9,118 @@ Requirements
...
+
+**NOTE:** Before you start work on a feature, please make sure to check the
+[issue tracker][gh-issues] and existing [pull requests][gh-prs] to ensure that
+work is not being duplicated. For further clarification, you can also ask in a
+new issue.
+
+[gh-issues]: https://github.com/terraform-providers/terraform-provider-vmc/issues
+[gh-prs]: https://github.com/terraform-providers/terraform-provider-vmc/pulls
+
+If you wish to work on the provider, you'll first need [Go][go-website]
```
Comment (rchadalawada) [inline README.md]: This is repeated

File: README.md (hunk 6 above)
Comment (saloneerege) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (saloneerege) [reply README.md]: Removed

**Review by rchadalawada — APPROVED** [review]

**Review by shylasrinivas — APPROVED** [review]

**Pull request merged**